packages/bpk-component-infinite-scroll/stories.js

```diff
@@ -93,11 +105,43 @@ storiesOf('bpk-component-infinite-scroll', module)
       });
     return (
       <InfiniteList
-        onItemsFetch={delayed}
+        dataSource={new MyDS(delayed)}
         elementsPerScroll={10}
         renderLoadingComponent={() => (
           <BpkSpinner type={SPINNER_TYPES.primary} />
         )}
       />
     );
+  })
+  .add('Force update data', () => {
+    class StoryDataSource extends DataSource {
```

Comment:
Can we move class definition outside this function? I think if we implement the `ArrayDataSource` we can add the `updateData` method there too.

Reply:
done